index.js

```diff
@@ -1,185 +1,232 @@
+  "BTCZ": 0x800000b1,
+  "BTF": 0x800026a0,
+  "BTG": 0x8000009c,
+  "BTM": 0x80000099,
+  "BTN": 0x800003e8,
+  "BTP": 0x80002327,
+  "BTQ": 0x80002093,
+  "BTV": 0x8000270f,
+  "BTW": 0x80000309,
+  "BTX": 0x800000a0,
+  "BURST": 0x8000001e,
   "BUZZ": 0x800000a9,
-  "Bata": 0x80000059,
-  "BelaCoin": 0x80000049,
-  "BigUp": 0x80000064,
-  "Biocoin": 0x8000009a,
-  "BitBean": 0x80000058,
-  "BitSend": 0x8000005b,
-  "BitZeny": 0x8000007b,
-  "Bitcoin": 0x80000000,
-  "Bitcoin 2x": 0x8000009d,
-  "Bitcoin Cash": 0x80000091,
-  "Bitcoin Diamond": 0x80000643,
-  "Bitcoin Gold": 0x8000009c,
-  "Bitcoindark": 0x80000034,
-  "Bitcoinplus": 0x80000041,
-  "Bitcore": 0x800000a0,
-  "Blackcoin": 0x8000000a,
-  "Breakout Coin": 0x8000007e,
-  "Bridgecoin": 0x800000a2,
-  "BritCoin": 0x80000046,
-  "Burst": 0x8000001e,
-  "Bytom": 0x80000099,
-  "Canada eCoin": 0x80000022,
-  "Cannacoin": 0x80000013,
-  "Cardano": 0x80000717,
-  "Chronos": 0x80000060,
-  "Clams": 0x80000017,
-  "ClearingHouse": 0x8000000f,
-  "ClubCoin": 0x8000004f,
-  "Compcoin": 0x8000004a,
-  "Counterparty": 0x80000009,
-  "Crown": 0x80000048,
-  "Dash": 0x80000005,
-  "Decred": 0x8000002a,
-  "Defcoin": 0x80000539,
-  "Denarius": 0x80000074,
-  "Diamond": 0x80000098,
-  "DigiByte": 0x80000014,
-  "Digitalcoin": 0x80000012,
-  "Dogecoin": 0x80000003,
-  "Dollarcoins": 0x80000066,
-  "Dopecoin": 0x80000035,
-  "Dubaicoin": 0x80000068,
-  "EDRCoin": 0x80000038,
-  "Ecoin": 0x80000073,
-  "Einsteinium": 0x80000029,
-  "Electronic Gulden": 0x8000004e,
-  "Ellaism": 0x800000a3,
-  "Ember": 0x800000aa,
-  "Ether": 0x8000003c,
-  "Ether Classic": 0x8000003d,
-  "EuropeCoin": 0x80000097,
-  "Evotion": 0x80000062,
-  "Expanse": 0x80000028,
-  "Factom Entry Credits": 0x80000084,
-  "Factom Factoids": 0x80000083,
-  "Feathercoin": 0x80000008,
-  "Firstcoin": 0x800000a7,
-  "Flashcoin": 0x80000078,
-  "FujiCoin": 0x8000004b,
-  "GameCredits": 0x80000065,
-  "Giftblock": 0x8000008a,
-  "Global Currency Reserve": 0x80000031,
-  "GoldBlocks": 0x8000005e,
-  "Gridcoin": 0x80000054,
-  "Groestlcoin": 0x80000011,
-  "Gulden": 0x80000057,
-  "HTMLCOIN": 0x800000ac,
-  "Hcash": 0x800000ab,
-  "Helleniccoin": 0x800000a8,
-  "Hempcoin": 0x80000071,
-  "IXCoin": 0x80000056,
-  "InsaneCoin": 0x80000044,
-  "Internet of People": 0x80000042,
-  "Jumbucks": 0x8000001a,
-  "Komodo": 0x8000008d,
-  "LBRY Credits": 0x8000008c,
-  "Landcoin": 0x8000003f,
-  "Libertas": 0x8000002e,
-  "Link": 0x8000004c,
-  "Linx": 0x80000072,
-  "Lisk": 0x80000086,
-  "Litecoin": 0x80000002,
+  "CCN": 0x80000013,
+  "CDN": 0x80000022,
+  "CDY": 0x80000479,
+  "CLAM": 0x80000017,
+  "CLUB": 0x8000004f,
+  "CMP": 0x80000047,
+  "CRAVE": 0x800000ba,
+  "CRW": 0x80000048,
+  "CRX": 0x80000060,
+  "DBIC": 0x80000068,
+  "DCR": 0x8000002a,
+  "DFC": 0x80000539,
+  "DGB": 0x80000014,
+  "DGC": 0x80000012,
+  "DLC": 0x80000066,
+  "DMD": 0x80000098,
+  "DNR": 0x80000074,
+  "DOGE": 0x80000003,
+  "DOPE": 0x80000035,
+  "DSH": 0x80000005,
```

Comment:
Why not `DASH`? Why this odd abbreviation?

Reply:
It should be `DASH`, this is incorrect.

Reply:
Ah, but it's incorrect on the list itself: https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
Yeah, so we can come back to this when they change it in the official repo. Thanks for checking, @jprichardson!